### PR TITLE
Feature/skeleton loader

### PR DESCRIPTION
**Pull request by priyankatechneai**
Card skeleton add in ConsolidatedView  master

### CODE REVIEW

**Review by AmitSolnke — CHANGES_REQUESTED** [review]
Kindly check the responsiveness from all aspects.
cards are overlapping on each other.

**Comment by priyankatechneai** [comment]
resolved

**Comment by priyankatechneai** [comment]
resolved

File: src/components/custom/loader/CardLoadingSkeleton.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <div className="container">
+      {[...new Array(rowCount)].map((_, rowIndex) => (
+        <div key={rowIndex} className="row mb-4">
+          {[...new Array(3)].map((_, colIndex) => (
+            <div key={colIndex} className="col-md-4">
+              <div className="card">
+                <div className="card-body">
+                  <div
+                    className="skeleton skeleton-text mb-3"
+                    style={{ width: "80%", height: "20px" }}
```
Comment (somyaranjan-techneai) [inline src/components/custom/loader/CardLoadingSkeleton.jsx:14]: Remove all inline styles, use bootstrap class or external style

File: src/screens/ConsolidatedView/ConsolidatedSlice.js [hunk 2]
```diff
@@ -13,19 +16,20 @@ export const ConsolidatedSlice = createSlice({
...
     },
   },
   extraReducers: (builder) => {
     builder.addCase(consolidatedData.pending, (state) => {
+      state.isLoading.consolidatedDataList = true;
       state.status = "loading";
     });
     builder.addCase(consolidatedData.fulfilled, (state, action) => {
       const { payload } = action;
+      state.isLoading.consolidatedDataList = false;
```
Comment (somyaranjan-techneai) [inline src/screens/ConsolidatedView/ConsolidatedSlice.js:28]: don't use unnecessary state, use existing state

File: src/screens/ConsolidatedView/CurrentClientProject.jsx [hunk 3]
```diff
@@ -7,18 +7,16 @@
 // import ListGroup from "react-bootstrap/ListGroup";
 // import ConsolidatedService from '../../services/ProjectManagementService/ConsolidatedService'
 
-
 // export default function CurrentClientProject(props) {
- 
+
```
Comment (somyaranjan-techneai) [inline src/screens/ConsolidatedView/CurrentClientProject.jsx:11]: Remove commented code, if not requard

File: src/screens/ConsolidatedView/CurrentClientProject.jsx [hunk 4]
```diff
@@ -122,135 +120,158 @@
...
+    "bg-lightyellow",
+    "light-success-bg",
+    "light-orange-bg",
+    "bg-lightblue",
+    "bg-lightgreen",
+    "light-success-bg",
+    "light-orange-bg",
+  ];
+
+  useState(() => {}, []);
```
Comment (somyaranjan-techneai) [inline src/screens/ConsolidatedView/CurrentClientProject.jsx:154]: what is this

File: src/components/custom/loader/CardLoadingSkeleton.jsx (hunk 1 above)
Comment (priyankatechneai) [reply src/components/custom/loader/CardLoadingSkeleton.jsx:14]: resolved

File: src/screens/ConsolidatedView/ConsolidatedSlice.js (hunk 2 above)
Comment (priyankatechneai) [reply src/screens/ConsolidatedView/ConsolidatedSlice.js:28]: resolved

File: src/screens/ConsolidatedView/CurrentClientProject.jsx (hunk 3 above)
Comment (priyankatechneai) [reply src/screens/ConsolidatedView/CurrentClientProject.jsx:11]: resolved

File: src/screens/ConsolidatedView/CurrentClientProject.jsx (hunk 4 above)
Comment (priyankatechneai) [reply src/screens/ConsolidatedView/CurrentClientProject.jsx:154]: resolved

**Pull request merged**